src/App.vue

```diff
@@ -1,56 +1,33 @@
 <template>
   <v-app>
-    <v-app-bar app color="primary" dark>
-      <div class="d-flex align-center">
-        <v-img
-          alt="Vuetify Logo"
-          class="shrink mr-2"
-          contain
-          src="https://cdn.vuetifyjs.com/images/logos/vuetify-logo-dark.png"
-          transition="scale-transition"
-          width="40"
-        />
-
-        <v-img
-          alt="Vuetify Name"
-          class="shrink mt-1 hidden-sm-and-down"
-          contain
-          min-width="100"
-          src="https://cdn.vuetifyjs.com/images/logos/vuetify-name-dark.png"
-          width="100"
-        />
-      </div>
-
-      <v-spacer></v-spacer>
-
-      <v-btn
-        href="https://github.com/vuetifyjs/vuetify/releases/latest"
-        target="_blank"
-        text
-      >
-        <span class="mr-2">Latest Release</span>
-        <v-icon>mdi-open-in-new</v-icon>
-      </v-btn>
-    </v-app-bar>
-
+    <Header />
     <v-main>
-      <HelloWorld />
+      <status-overlay v-if="status !== 'Success'" v-bind:status="status" />
+      <router-view v-else />
     </v-main>
+    <Footer />
   </v-app>
 </template>
 
 <script>
-import HelloWorld from "./components/HelloWorld";
+import Footer from "./components/Footer.vue";
+import Header from "./components/Header.vue";
+import StatusOverlay from "./components/StatusOverlay.vue";
 
 export default {
   name: "App",
-
   components: {
-    HelloWorld
+    Footer,
+    StatusOverlay,
+    Header
   },
-
-  data: () => ({
-    //
-  })
+  created() {
+    this.$store.dispatch("getData");
+  },
+  computed: {
+    status() {
+      return this.$store.getters.getStatus;
+    }
```

Comment:
I thought that mapGetters for just one getter is a bit overkill (it's a small project, so it won't be a performance issue, but wouldn't it increase complexity by mapping over an array instead of just calling one method?)

Reply:
I don't think it would. You can check the result code but it may even be inlined to the very same JS code. And even if it's not, it won't really matter, as the cost of iterating over an array at runtime (even if it is at runtime) is miniscule compared to the cost of creating a method.